src/backend/joanie/tests/test_models_course.py

```diff
@@ -3,13 +3,17 @@
 """
 from django.core.exceptions import ValidationError
 from django.test import TestCase
+from django.utils.translation import activate, deactivate
 
 from joanie.core import factories, models
 
 
 class CourseRunModelsTestCase(TestCase):
     """Test suite for the CourseRun model."""
 
+    def tearDown(self):
+        deactivate()
```

Comment:
better than this is to use a context manager to activate the language:
https://github.com/openfun/richie/blob/master/tests/apps/courses/test_models_blogpost.py#L142